confluent/docker_utils/cub.py

```diff
@@ -94,15 +99,29 @@ def __request(host, port, secure, ignore_cert, username='', password='', path=''
     auth = HTTPBasicAuth(username, password) if (username or password) else None
     return requests.get(url, verify = not ignore_cert, auth = auth)
 
-def log4j_config_file():
-    config_file = DEFAULT_LOG4J_FILE
+
+def use_log4j2():
+    return os.environ.get("USE_LOG4J_2", None) is not None
```

Comment:
hey, do we want to explicitly check for boolean value - True/ False instead of non None?